lib/Doctrine/Common/DataFixtures/Purger/ORMPurger.php

```diff
@@ -130,7 +130,7 @@ public function purge()
         }
 
         $connection            = $this->em->getConnection();
-        $filterExpr            = $connection->getConfiguration()->getFilterSchemaAssetsExpression();
+        $filterExpr            = method_exists($connection->getConfiguration(), 'getFilterSchemaAssetsExpression') ? $connection->getConfiguration()->getFilterSchemaAssetsExpression() : null;
```

Comment:
Please use linebreaks :pray:

Reply:
I think line breaks will not be coherent with the coding style of this file and I'm not sure it improves readability.
It would look like this:
```php
        $connection            = $this->em->getConnection();
        $filterExpr            = method_exists($connection->getConfiguration(), 'getFilterSchemaAssetsExpression')
            ? $connection->getConfiguration()->getFilterSchemaAssetsExpression()
            : null;
        $emptyFilterExpression = empty($filterExpr);

        $schemaAssetsFilter = method_exists($connection->getConfiguration(), 'getSchemaAssetsFilter') ? $connection->getConfiguration()->getSchemaAssetsFilter() : null;
```

Maybe formatting can be delegated to a separate PR?

Reply:
The coding style is really defined at the organization level, not at the file level

Breaking inside `method_exists` would result in a better distribution (in your example, the first line still requires horizontal scrolling).

```php
$connection            = $this->em->getConnection();
$filterExpr            = method_exists(
    $connection->getConfiguration(),
    'getFilterSchemaAssetsExpression'
) ? $connection->getConfiguration()->getFilterSchemaAssetsExpression() : null;
$emptyFilterExpression = empty($filterExpr);

$schemaAssetsFilter = method_exists(
    $connection->getConfiguration(),
    'getSchemaAssetsFilter'
) ? $connection->getConfiguration()->getSchemaAssetsFilter() : null;
```

> Maybe formatting can be delegated to a separate PR?

Sure! https://github.com/doctrine/data-fixtures/pull/367